source/common/router/router.h

```diff
@@ -616,10 +616,11 @@ class Filter : Logger::Loggable<Logger::Id::router>,
   // mapped to the runtime guard "envoy.restart_features.ensure_connection_retry"
   // on router creation.
   bool ensure_connection_retry_ : 1;
-  // Set to true when the router is waiting to get an upcoming connection from the upstream,
-  // this will be set to true if ensure_connection_retry_ above is true and the router did
-  // not get a connection.
-  bool wait_for_connect_ : 1;
+  // Set to true when ensure_connection_retry_ above is true, retry on connection-failure is
+  // enabled, and the router is waiting to get an upcoming connection from an upstream.
+  // This will potentially increase the limit to which we are willing to buffer in order to ensure
+  // we retry the connection failure.
```

Comment:
so if the plan is to limit when this is set to not only be when it's waiting on upstream connection this isn't a good name.
What is this really supposed to be tracking? Is it the fact the buffer has an increased limit?  Let's name it for what it is.

Reply:
Right, this is tracking the fact that we temporary increase the retry shadow buffer limit until we got a conneciton.
renamed it: `increased_retry_shadow_buffer_limit_until_got_connection_`